flang/test/Driver/emit-llvm-all-drivers.f90

```diff
@@ -0,0 +1,25 @@
+! RUN: %flang_fc1 -emit-llvm %s -o - | FileCheck %s
+! RUN: %flang_fc1 -emit-mlir %s -o - | tco | FileCheck %s
+! RUN: bbc %s -o - | tco -o - | FileCheck %s
+
+! This is added as a regression test for a failure reported in
+! https://github.com/flang-compiler/f18-llvm-project/issues/1280. It makes sure
+! that this example can be lowered to LLVM IR by all major drivers in Flang.
+! Ultimately, we should be veryfying that the generated code is identical in
+! all the tested cases. Currently, there are some discrepancy in the the
+! generated debug info.
```

Comment:
Nit: there `is` some discrepancy in `the` generated
or
Nit: there `are` some discrepancies in `the` generated